docs/src/components/propsTable/PropsTableBody.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { remark } from 'remark';
+import html from 'remark-html';
+import { Badge, TableRow, View } from '@aws-amplify/ui-react';
+import { ResponsiveTableCell } from '@/components/ResponsiveTable';
+import { CodeHighlight } from '@/components/CodeHighlight';
+import { Property } from '../../../scripts/types/catalog';
+
+export const PropsTableBody = ([
+  propName,
+  { name, type, description, isOptional },
+]: [string, Property]): JSX.Element => {
+  const [htmlDescription, setHtmlDescription] = useState('-');
+
+  useEffect(() => {
+    const getDesc = async () => {
+      const descriptionHTML = await remark().use(html).process(description);
+      setHtmlDescription(descriptionHTML.toString());
+    };
+    getDesc();
+  }, [description]);
+
+  return (
+    <TableRow key={name}>
+      <ResponsiveTableCell label="Name">
+        {name} {` `}
+        {isOptional ? null : (
+          <Badge variation="info" size="small">
+            Required
+          </Badge>
+        )}
+      </ResponsiveTableCell>
+      <ResponsiveTableCell label="Type">
+        <CodeHighlight code={type} />
```

Comment:
```suggestion
        <CodeHighlight code={type} language="typescript" />
```

This should add the color syntax highlighting back if we want that.

Reply:
For above comment^ <img width="300" alt="Screen Shot 2022-11-16 at 1 21 31 PM" src="https://user-images.githubusercontent.com/376920/202261923-ce872093-88be-409d-819b-1a3499a5a892.png">